cli/.js/src/main/scala/aqua/run/RunCommand.scala

```diff
@@ -81,19 +85,8 @@ object RunCommand extends Logging {
             .toFuture
           _ = OutputPrinter.print("Your peerId: " + peer.getStatus().peerId)
           promise = Promise.apply[Unit]()
-          _ = CallJsFunction.registerUnitService(
-            peer,
-            config.consoleServiceId,
-            config.printFunctionName,
-            args => {
-              val str = JSON.stringify(args, space = 2)
-              // if an input function returns a result, our success will be after it is printed
-              // otherwise finish after JS SDK will finish sending a request
-              // TODO use custom function for output
-              OutputPrinter.print(str)
-              promise.success(())
-            }
-          )
+          _ = consoleService.registerService(peer, promise)
+          _ = config.services.values.map(_.registerService(peer))
```

Comment:
Let's rename `config.services` to `config.fn_arguments`

Reply:
i will rename it as `argumentGetters`